target/linux/x86/image/grub-efi.cfg

```diff
@@ -6,7 +6,13 @@ set timeout="@TIMEOUT@"
 search -l kernel -s root
 
 menuentry "@TITLE@" {
-	linux /boot/vmlinuz @GPT_ROOTPART@ @CMDLINE@ noinitrd
+	linux /boot/vmlinuz @GPT_ROOTPART@ @CMDLINE@
+	if [ -s /boot/intel-ucode.img ]; then
+		initrd /boot/intel-ucode.img
+	fi
+	if [ -s /boot/amd-ucode.img ]; then
+		initrd /boot/amd-ucode.img
+	fi
```

Comment:
To put it simply:
```
[ -s /boot/intel-ucode.img ] && initrd /boot/intel-ucode.img
[ -s /boot/amd-ucode.img ] && initrd /boot/amd-ucode.img
```

Reply:
I can change it isn't this just a stylistic change?

Reply:
It's just a stylistic change.

I wonder what will happen if there are intel-ucode.img and amd-ucode.img files. Is that even possible?

Reply:
> I wonder what happens if there are both intel-ucode.img and amd-ucode.img files. Is that even possible?

Treat these as simple firmware blobs which are copied to RAM instead having them on root fs, so kernel can load them whenever. Kernel matches the microcode with CPU it's running on, so no conflicts can happen.

Reply:
> It's just a stylistic change.

I made the change